.changes/next-release/Bug Fix-b28584d3-e664-4851-9d25-0949db6ca0ac.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Amazon Q: Fixed prompt input submit with enter if IME enabled (issues/4125), requires shift+enter while IME enabled"
```

GitHub will hyperlink the changelog.

```suggestion
	"description": "Amazon Q: Fixed prompt input submit with enter if IME enabled [#4125](https://github.com/aws/aws-toolkit-vscode/issues/4125), requires shift+enter while IME enabled"
```